### ISSUE TITLE
2dos

### ISSUE THREAD

**Issue by baconstrudel** [ascavis/machine-magic#2]
-figure out a concept for how the data is preprocessed automatically
-settle for a couple of different Networks to try
-implement Networks
-Magic the data into beautifully categorized asteroids

**Comment by seyr** [comment]
I take it you have already decided on the algorithm (class)? Neural networks? Can you give me the name/overview link maybe so I can get up to speed?

**Comment by baconstrudel** [comment]
ok so the inital Approach would be a deep belief network with dropout and pretraining

**Comment by baconstrudel** [comment]
second approach would be a 2 step approach, the first step being a dimension reduction with a sanger-net or kohonen-map and then a perceptron for classification, maybe a two layer-network with RBM and perceptron

**Comment by baconstrudel** [comment]
rbm = Restricted Boltzmann machine

**Comment by baconstrudel** [comment]
I meant neural gas not sanger-net

**Comment by baconstrudel** [comment]
how do you get along?

**Comment by seyr** [comment]
Still reading. Did you already start with code?

**Comment by baconstrudel** [comment]
I think we might want to do it together

**Comment by seyr** [comment]
I see. You had libraries yesterday, no? Did you get my messages? (dragon.fly@jabber.org)

**Comment by seyr** [comment]
(have to get some food now, I'll be back)
